al-khaser/AntiAnalysis/process.cpp

```diff
@@ -33,7 +33,21 @@ VOID analysis_tools_process()
 		_T("sniff_hit.exe"),		// Part of SysAnalyzer iDefense
 		_T("windbg.exe"),			// Microsoft WinDbg
 		_T("joeboxcontrol.exe"),	// Part of Joe Sandbox
-		_T("joeboxserver.exe")		// Part of Joe Sandbox
+		_T("joeboxserver.exe"),		// Part of Joe Sandbox
+		_T("HxD.exe"),				// HxD - Freeware Hex Editor and Disk Editor
+		_T("ResourceHacker.exe"),	// Resource Hacker
+		_T("httpdebugger.exe"),		// Http Debugger
+		_T("windowrenamer.exe"),	// Window Renamer
```

Comment:
And same for windowrenamer? I think it is a legit app, not related to RCE ?

Reply:
Why not?

Reply:
IMHO, it is not a reverse engineering tool, neither an automated sandboxing system. @gsuberland @hfiref0x what do you think ?

Reply:
Never heard about that windowrenamer. Imho is better add something like "fiddler" instead.

Reply:
Thanks, I would say replace windowsrenamer with fiddler and take out the VS entries. Because these tools are not RE/Sandboxing processes.

Reply:
Agreed. VS entries are not strongly indicative of analysis. Never looked at WindowRenamer.